### ISSUE TITLE
soundVolumeIncreasePercentage not working as intended

### ISSUE THREAD

**Issue by Der-Matze** [modelleicher/FS22_simpleIC#29]
Hi, just a quick question: Is it possible that soundVolumeIncreasePercentage is not implemented yet and has no impact on the indoor sound? 

Btw: @modelleicher  Danke für diesen großartigen Mod. Er eröffnet so viele neue Möglichkeiten!

**Comment by modelleicher** [comment]
Yep thats not implemented yet because I didn't get into the new sound system yet

**Comment by GrandSurf** [comment]
Any news for implemented soundVolumeIncreasePercentage?
